src/main/resources/META-INF/rewrite/ibm-java.yml

```diff
@@ -110,4 +112,38 @@ recipeList:
   - org.openrewrite.java.ChangeType:
       oldFullyQualifiedTypeName: com.ibm.security.auth.module.Krb5LoginModule
       newFullyQualifiedTypeName: com.sun.security.auth.module.Krb5LoginModule
-      ignoreDefinition: true
\ No newline at end of file
+      ignoreDefinition: true
+---
+type: specs.openrewrite.org/v1beta/recipe
+name: org.openrewrite.java.migrate.RemovedJavaXMLWSModuleProvided
+displayName: Do not use `java.xml.ws` module in application
```

Comment:
Let's update the name to `Do not use `java.xml.ws` module in WebSphere Liberty`

Reply:
updated in next commit